src/pages/home/home.component.tsx

```diff
@@ -0,0 +1,19 @@
+import {signOutUser} from "../../utils/firebase/firebase.utils"
+import {useNavigate} from "react-router-dom"
+
+function HomePage() {
+	const navigate = useNavigate()
+	const signOutHandler = () => {
+		const resp = signOutUser()
```

Comment:
It is not advised to use these kind of shortcuts for variable names. Better to rename it to `response`

Reply:
Fixed